src/parse_options.c

```diff
@@ -433,6 +433,7 @@ option_help * createGlobalOptions(IOR_param_t * params){
     {'y', NULL,        "dualMount -- use dual mount points for a filesystem", OPTION_FLAG, 'd', & params->dualMount},
     {'Y', NULL,        "fsyncPerWrite -- perform sync operation after every write operation", OPTION_FLAG, 'd', & params->fsyncPerWrite},
     {'z', NULL,        "randomOffset -- access is to random, not sequential, offsets within a file", OPTION_FLAG, 'd', & params->randomOffset},
+    {0, "random-offset-seed",        "The seed for -z", OPTION_OPTIONAL_ARGUMENT, 'd', & params->randomSeed},
```

Comment:
Is this a part of the md-workbench patch or is this something that got accidentally rolled into this PR?

Reply:
You are right, I used it for testing. It is the "random" patch that I cherry-picked for the random branch. Merge should be conflict-free and this extra option doesn't do anything if not set  (adds reproducibility if set).